pallets/portfolio/src/lib.rs

```diff
@@ -516,6 +516,16 @@ impl<T: Trait> Module<T> {
 
         Ok(())
     }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    pub fn fund_portfolio(
```

Comment:
IMO this should be part of the benchmarking file, not the portfolio's lib.rs

Reply:
https://github.com/PolymathNetwork/Polymesh/pull/726#discussion_r520513304